src/components/structures/ContextMenu.tsx

```diff
@@ -389,7 +389,7 @@ export class ContextMenu extends React.PureComponent<IProps, IState> {
     }
 
     render(): React.ReactChild {
-        return ReactDOM.createPortal(this.renderMenu(), getOrCreateContainer());
+        return ReactDOM.createPortal(this.renderMenu(), document.body);
```

Comment:
we mount/unmount using `getOrCreateContainer` elsewhere in the context menu code - what is this change accomplishing?

Reply:
It is pretty much to reduce the use extra nodes where they are not needed. Most of the time that container is not really required

Reply:
not sure I fully understand: we are still referencing `getOrCreateContainer()` within the class and outside of `render()`: is this supposed to be fixed too?